package.json

```diff
@@ -6,19 +6,41 @@
   "license": "MIT",
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "lerna run libbuild --stream",
-    "build-storybook": "lerna run libbuild --stream --scope=@cmpsr/components && lerna run build-storybook --stream",
-    "test": "lerna run test --stream",
-    "test:watch": "lerna run --parallel test:watch --stream",
-    "test:coverage": "lerna run --parallel test:coverage --stream",
+    "build": "turbo run libbuild",
+    "build-storybook": "turbo run build-storybook",
+    "test": "turbo run test",
+    "test:watch": "turbo run test:watch --parallel",
+    "test:coverage": "turbo run test:coverage",
     "lint": "eslint . --ext .tsx,.ts,.js",
     "lint-and-fix": "eslint . --ext .tsx,.ts,.js --fix",
-    "publish-ci": "lerna run libbuild --stream && lerna publish from-package -y --no-verify-access"
+    "publish-ci": "turbo run libbuild && lerna publish from-package -y --no-verify-access"
```

Comment:
do you have an idea about how feasible is moving the release process from learna to turbo?

Reply:
turbo is not meant to be a release tool. We could do this in another tool however.

Reply:
from their docs

> tl;dr you can use Lerna and Turbo together.

> Use lerna symlinking (lerna bootstrap) packages, publishing and changelog generation
> Use turbo for task running and caching